.goreleaser.yml

```diff
@@ -77,3 +80,19 @@ dockers:
     - "sonatypecommunity/nancy:{{ .Tag }}"
     - "sonatypecommunity/nancy:v{{ .Major }}"
     - "sonatypecommunity/nancy:v{{ .Major }}.{{ .Minor }}"
+  -
+    goos: linux
+    goarch: amd64
+    dockerfile: build/Dockerfile.ci
+    binaries:
+    - nancy
+    build_flag_templates:
+    - "--pull"
+    - "--label=author='DJ Schleen'"
+    - "--label=version={{ .Version }}"
+    - "--label=maintainer='sonatype-nexus-community'"
+    image_templates:
+    - "sonatypecommunity/nancy:ci"
+    - "sonatypecommunity/nancy:ci-{{ .Tag }}"
+    - "sonatypecommunity/nancy:ci-v{{ .Major }}"
+    - "sonatypecommunity/nancy:ci-v{{ .Major }}.{{ .Minor }}"
```

Comment:
Same question about `v` or not to `v` here too.

Reply:
`{{ .Version }}` shouldn't have the `v` though. Major, Minor doesnt' have it for sure.

https://goreleaser.com/customization/#Name%20Templates